myapp/src/day4/Question7/SearchFilter.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react'
+import './styles.css'
+
+const SearchFilter = () => {
+  const [items, setItems] = useState([])
+  const [searchInput, setSearchInput] = useState('')
+  const filterSearch = async () => {
+    const res = await fetch('https://dummyjson.com/products')
```

Comment:
get the data from API once and store it. simply filter values from it when user types anything. We don't need to hit API again

Reply:
done